btn-1cc-checkout.js

```diff
@@ -43,109 +43,113 @@ function btnCheckout(){
     });
   }
 
-  // For attaching event listener to Woodmart's sticky add-to-cart
-  document.addEventListener('scroll',(e)=>{
-     
-    var stickyBtn=document.querySelectorAll('#btn-1cc-pdp')[1];
-
-    let i = 0;
-     while (typeof quantity === 'undefined') {
-        var quantity = document.getElementsByClassName("qty")[i].value;
-        i++;
-     }
-
-    stickyBtn.setAttribute('quantity', quantity);
-
-      jQuery('.qty').on('change',function(e)
-     {
-        let x = 0;
-         while (typeof quantity === 'undefined') {
-          var quantity = document.getElementsByClassName("qty")[x].value;
-          x++;
-         }
-
-        stickyBtn.setAttribute('quantity', quantity);
+  var stickyBtn = document.querySelectorAll('#btn-1cc-pdp')[1];
```

Comment:
@angbpy This id will only have the value in a single product page and in case of a sticky button is not available, will that log any error on the console? Please verify it.

Reply:
No errors as we are not dereferencing it without null safety check now